components/datetime/tests/fixtures/tests/components-partial-matches.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "description": "Partial match for m -> hm -> h:m a",
+        "description": "Singleton fields are returned directly if there are no singleton matches (m -> hm)",
```

Comment:
suggestion: It would be good to have a bit better code coverage here with:

* Additional locales
* Additional field types
* Additional field widths.